src/main/java/dinkplugin/notifiers/CollectionNotifier.java

```diff
@@ -49,12 +62,25 @@ private void handleNotify(String itemName) {
             new String[] { Utils.getPlayerName(client), itemName }
         );
 
+        // read updated # of collection log entries completed from varplayer id's
+        int completed = client.getVarpValue(COMPLETED_VARP);
+        int total = client.getVarpValue(TOTAL_VARP); // unique; doesn't over-count duplicates
+        boolean varpValid = total > 0 && completed >= 0;
```

Comment:
```suggestion
        boolean varpValid = total > 1000 && completed > 0;
```
if we want to be more robust to a scenario where jagex changes the varpId

Reply:
Since the consequence of this failure would just be that the value of `completed` & `total` is wrong, I'd even go so far as to not validate this at all.

If we don't do validation, the value will always flow through and we'll be quickly notified that something is wrong if comparing the completion/total with the chatlog entry in the screenshot.

Reply:
We do need *some* validation to avoid divide by zero. My hope is people would get used to the completed field & report if it went missing one day. Perhaps we can add a log.warn on invalid varp

Reply:
Right, makes sense since we send the completion % along, I think leaving it as is with minimal validation is good.